opm/simulators/wells/WellState.cpp

```diff
@@ -695,9 +695,6 @@ void WellState<Scalar>::initWellStateMSWell(const std::vector<Well>& wells_ecl,
     for (int w = 0; w < nw; ++w) {
         const auto& well_ecl = wells_ecl[w];
         auto& ws = this->well(w);
-        // If the phase_rates has zero size this is an inactive well that will never be solved
-        if (ws.perf_data.phase_rates.size() == 0)
-            continue;
```

Comment:
Is this  a general bugfix independent of the changes in this and the earlier PR?

Reply:
No, I unfortunately introduced this in the earlier PR to counteract the side-effects of my previous misunderstanding.. Sorry for the confusion, and thanks for the swift handling of this bugfix!

Reply:
Yeah, doesn't really speak for me if you say 4 days is swift. :sweat_smile: 
Sorry. I was a little occupied with other stuff.